query/query4_test.go

```diff
@@ -1564,3 +1564,51 @@ func TestNumUids(t *testing.T) {
 	require.Equal(t, metrics.NumUids["name"], uint64(16))
 	require.Equal(t, metrics.NumUids["_total"], uint64(26))
 }
+
+func TestJaegerTag(t *testing.T) {
+	query := `
+		 {
+			 #tag:test
+			 me(func: uid(0x01)) {
+				 uid
+			 }
+		 }
+	 `
+	js := processQueryNoErr(t, query)
+	require.JSONEq(t,
+		`{"data": {"me":[ {"uid": "0x1"}]}}`,
+		js)
+}
+
+func TestJaegerTag1(t *testing.T) {
+	query := `
+		 {
+			 me(func: uid(0x01)) {
+				 uid
+				 #tag:test
+			 }
+		 }
+	 `
+	js := processQueryNoErr(t, query)
+	require.JSONEq(t,
+		`{"data": {"me":[ {"uid": "0x1"}]}}`,
+		js)
+}
+
+func TestJaegerTag2(t *testing.T) {
+	query := `
+		 {
+			 me(func: has(<#tag:test>)) {
+				 uid
+				 <#tag:test>
```

Comment:
same question here.

Reply:
For this one. You can see in `query/common_test.go` that I added the predicate `<#tag:test>`. This is captured by the logic as the function uses `"strings.Contains(m, "#tag")" `, but it doesn't cause any side effects. I consider this query to be okay. It's rare for someone to use this pattern in predicates, and even if it does happen, it's not an issue, its effect is innocuous.

But yes. This will be captured. As the hashtag along with the name "tag" and the colon is tha pattern that I'm looking for.